src/pkg/services/m365/api/graph/errors.go

```diff
@@ -138,6 +141,7 @@ var (
 	ErrResourceOwnerNotFound = clues.New("resource owner not found in tenant")
 
 	ErrTokenExpired = clues.New("jwt token expired")
+	ErrTokenInvalid = clues.New("jwt token invalid")
```

Comment:
It's currently standard to keep error sentinals around just in case.  Many of the other sentinels are likewise unused.  If we want to clean that up, a tech deb ticket is a better way to track the goal.

Reply:
It was introduced in this PR and not being used anywhere. Not sure about the value-add here for keeping it around.

Reply:
Still maintaining a standard.  I'm going to keep it around for this PR, but will follow up with a cleanup afterward.

Reply:
Thanks. Could you please remove it from `IsErrUnauthorizedOrBadToken` atleast? It's an additional dead conditional which is confusing because it's not being set anywhere.

Reply:
I'm standing by my statement.  Not to worry, it won't be around long.